liquidprompt

```diff
@@ -1261,7 +1288,7 @@ _lp_python_env() {
 _lp_python_env_color() {
     _lp_python_env || return "$?"
 
-    lp_python_env_color="[${LP_COLOR_VIRTUALENV}${lp_python_env}${NO_COL}]"
+    lp_python_env_color="${LP_COLOR_VIRTUALENV}${lp_python_env}${NO_COL}"
```

Comment:
Each one of these changes really should have a `.versionchanged: 2.2` note added to their documentation saying that they no longer include the `[...]`, as they are intended to be used with `_lp_dev_env_color()`.